mpich2/meta.yaml

```diff
@@ -1,13 +1,22 @@
-# This is legacy recipe, which has not been tested using conda-build.
-# See: https://github.com/ContinuumIO/anaconda-recipes/blob/master/LEGACY.md
-
 package:
   name: mpich2
   version: 1.4.1p1
 
 source:
   fn: mpich2-1.4.1p1.tar.gz
   md5: b470666749bcb4a0449a072a18e2c204
+  url: http://www.mpich.org/static/downloads/1.4.1p1/mpich2-1.4.1p1.tar.gz
+
+build:
+  detect_binary_files_with_prefix : True
+  number: 1
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - libgcc-ng
```

Comment:
You do not need to identify `libgcc-ng` as a `requirements/build`.

Reply:
Currently if I do not do this I see an error about not being able to find -lquadmath. conda-build doesn't automatically install libgcc-ng. I've just added a comment about this in the yaml file.

Reply:
It does automatically install libstdc++-ng and libgfortran-ng though.